enterprise/app/code/code.tsx

```diff
@@ -617,7 +615,7 @@ export default class CodeComponent extends React.Component<Props> {
   }
 
   updateState(newState: Partial<State>, callback?: VoidFunction) {
-    this.setState(newState, () => {
+    this.setState(newState as State, () => {
```

Comment:
Surprised this cast is needed, I though it took State|Partial<State>|(a bunch of other stuff)

Reply:
👍